scripts/linters/custom_eslint_checks/rules/protractor-practices.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
```

Comment:
I have named this file protractor-practices as I think all protractor-related good practices checks should be in one file.